examples/qml_minimal/CMakeLists.txt

```diff
@@ -85,6 +85,9 @@ add_executable(${APP_NAME} cpp/main.cpp qml/qml.qrc)
 target_link_libraries(${APP_NAME} PRIVATE ${CRATE})
 # ANCHOR_END: book_cmake_executable
 
+# If we are using a statically linked Qt then we need to import any qml plugins
```

Comment:
should probably mention this in the book as well ....

Reply:
Added to the linking part of the example, the code comment is enough to explain it.